### ISSUE TITLE
Duplicate content on website

### ISSUE THREAD

**Issue by Ruffianlab1** [Difegue/DoujinSoft#63]
I have seen that some games have duplicates on the site. I believe the only thing which is different is the id of the microgame, but the contents of the game are an exact match. I'm guessing when someone makes a copy of a mio, it changes the id, and therefore is counted as a different game. What I would do is write a script to check every mio in the database and check only the data to determine which one are duplicate of each other

**Comment by Difegue** [comment]
Yeah this is pretty complex, I haven't really bothered doing dupe checking besides removing exact matches.  
It's pretty common even for exact copies of the same game to be different on a byte-to-byte basis, even just importing a .mio to a game and exporting it back will change the data slightly.  

You'd have to dig pretty deep at the AI/resources level of a .mio to make sure a game is an exact copy of another without having false positives.  
And even then, there's been multiple occurences of a creator uploading multiple/fixed versions of the same game that are 99% the same except for some logic changes; Those would still be treated as non-duplicates.  

tl;dr it's hard, I'll leave the issue open though in case someone ever wants to try it
